data/de/conferences.yml

```diff
@@ -1,4 +1,15 @@
 items:
+  -  
+    name: JakartaOne Livestream 2025
+    location: Canada
```

Comment:
We should probably indicate Virtual for the 2025 event, as even for EF staff not all staff is based in Canada, and there is no physical location tied to the event.

Reply:
@autumnfound, But they are all virtual events... so by that logic, we shouldn't list any countries. It doesn't even look like this field is read anywhere. I feel like we can remove the location field entirely. WDYT?

Reply:
I honestly agree. I don't think we use the location field anymore, so +1 to that.